components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -2900,6 +2904,14 @@ subroutine cloud_water_conservation(qc,qcnuc,dt,    &
 
    real(rtype) :: sinks, sources, ratio
 
+#ifdef SCREAM_CONFIG_IS_CMAKE
```

Comment:
Why all these checks on whether the config is CMake? Looks like if not cmake, a bunch of code is not executed. Is this going to work correctly if built with CIME?

Reply:
@jgfouca

Reply:
@bartgol the reason for these ifdefs is exactly so this code will still build with CIME. CIME builds will not have scream so we can't use stuff from micro_p3_iso_f.

Reply:
Oh, this file is in cam, I see.